stories/Accordion.stories.mdx

```diff
@@ -1,5 +1,6 @@
 import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
 import { html } from 'lit-html';
+import {PrincetonDesignSystem} from "../dist/jazz_behavior.js";
```

Comment:
Why does the jazz_behavior.js need to be imported here?  I am assuming you only need it for use within the story, but isn't that handled in the .storybook/preview-head.html file when StoryBook builds src/index.ts